manager/controllers/cves/details.go

```diff
@@ -45,17 +45,17 @@ func init() {
 
 // GetCveDetails represents CVE detail endpoint controller.
 //
-// @id GetExposedClusters
+// @id GetCveDetails
 // @summary CVE details
 // @Tags cves
 // @description Endpoint return details for given CVE
 // @accept */*
 // @produce json
 // @Param cve_name path  string true  "CVE name"
 // @router /cves/{cve_name} [get]
-// @success 200 {array}  base.Response{data=GetCveDetailsSelect}
-// @failure 404 {object} base.Error{error=base.ErrorDetail{detail=string,status=int}} "{cve_name} not found"
```

Comment:
What is the reason to remove `ErrorDetail` from here?

Reply:
`Error` struct includes the `ErrorDetail`, it's redundant to specify
also the `error=` key is wrong, it produces this:

<img width="227" alt="Snímek obrazovky 2022-05-09 v 16 12 16" src="https://user-images.githubusercontent.com/6650196/167429523-da7c883b-46bb-4933-9659-e180dbaa4ff2.png">

Reply:
perhaps it should be `error` and not `errors`, I'll fix this too

Reply:
added commit